python/cudf/cudf/core/column/categorical.py

```diff
@@ -1236,7 +1377,9 @@ def _create_empty_categorical_column(categorical_column, dtype):
     )
 
 
-def pandas_categorical_as_column(categorical, codes=None):
+def pandas_categorical_as_column(
+    categorical, codes=None
```

Comment:
Can we not type `categorical` and `codes` here?

Reply:
Not at the moment, but I added an explicit `ColumnLike` annotation for them, which is currently an alias for `Any`.